content/download/fedora.adoc

```diff
@@ -28,16 +38,5 @@ dnf install kicad
 If you don't have copr install with:
 
 ----
-yum install dnf-plugins-core
+dnf install dnf-plugins-core
 ----
-
-As a side note, the wxwidgets build in the copr build is forcing `--with=gtk2` on wxgtk, otherwise eeschema and the legacy canvas does not work properly. 
```

Comment:
Because it doesn't really apply or matter anymore. A gtk2 enabled wxwidgets package is now provided directly from Fedora repositories and it doesn't require the user to remove a gtk3 version of wxwidgets.

Also this is not specific to the Copr -- the Fedora package uses the same version of wxwidgets. In fact the package source are mostly the same, differing only in source tarballs.